src/app/shared/navbar/navbar.ts

```diff
@@ -19,6 +20,17 @@ const SECTIONS_KEYS = Object.keys(SECTIONS);
 })
 export class NavBar {
   isNextVersion = location.hostname.startsWith('next.material.angular.io');
+  skipLinkTarget: string;
+  skipLinkHidden = true;
+  constructor(router: Router) {
+    router.events
+      .pipe(filter((event: Event) => event instanceof NavigationEnd))
+      .subscribe(() => {
+        // set skip link
+        const baseUrl = router.url.split('#')[0];
+        this.skipLinkTarget = `${baseUrl}#main-content`;
```

Comment:
can we call this `skipLinkHref`, `target` is actually a different attribute on `<a>` tags that tells it whether to open in the same window or a new one